admin_ui_support/admin_ui_support.routing.yml

```diff
@@ -1,48 +1,4 @@
 # Routes controlled by the React App.
-admin_ui_support.user.permissions:
```

Comment:
This certainly removes the permissions page as well

Reply:
Maybe we can just remove the dblog component here and have an issue to update the permissions/roles pages to the material ui design?

Reply:
Sounds sensible for me!